packages/carequality-sdk/src/client/__tests__/carequality-fhir.test.ts

```diff
@@ -0,0 +1,230 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/no-non-null-assertion */
+import * as dotenv from "dotenv";
+dotenv.config();
+// Keep dotenv import and config before everything else.
+import { faker } from "@faker-js/faker";
+import { cloneDeep } from "lodash";
+import { APIMode, CarequalityManagementApi, OrganizationWithId } from "../carequality";
+import { CarequalityManagementApiFhirMock } from "./carequality-fhir-mock";
+import { getOrganizationFhirTemplate } from "./organization-template";
+import { getApiMode, getEnvVar, getEnvVarOrFail } from "./shared";
+
+jest.setTimeout(80_000);
+
+const orgOidNode = 5;
+
+function makeCarequalityManagementAPI(): CarequalityManagementApi | undefined {
+  const apiKey = getEnvVar("CQ_MANAGEMENT_API_KEY");
+  if (!apiKey) return undefined;
+  const apiMode = getApiMode();
+  return new CarequalityManagementApiFhirMock({
+    apiKey,
+    apiMode,
+  });
+}
```

Comment:
Is this what is known as defensive programming? 🤩 First time I'm seeing it XD. Nice!

Reply:
You might say that. 😄